### PR TITLE
Cleanup and UX fixes

### PR DESCRIPTION
**Pull request by ThislsAUsername**

### CODE REVIEW

File: src/UI/Art/SpriteArtist/InfoView.java [hunk 1]
```diff
@@ -77,22 +78,47 @@ public void render(Graphics g)
...
+
           // Build our overlay
-          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(drawingWidth, overlayHeight);
+          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(drawingWidth, coEntryHeight);
+          Graphics overlayG = overlayPic.getGraphics();
+
+          // Highlight the selected CO
+          if( i == coIndex )
+          {
+            overlayG.setColor(new Color(246, 234, 210));
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/InfoView.java]: Why not `SpriteUIUtils.MENUHIGHLIGHTCOLOR`?

File: src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java [hunk 2]
```diff
@@ -189,4 +190,20 @@ public BufferedImage update(PlayerSetupInfo info)
       return myImage;
     }
   }
+
+  /**
+   * Draws little white boxes around all the currently-highlighted player's properties.
+   * @param distance - defines how far the boxes should be from the property edge (serves to highlight the new properties when switching between players)
+   */
+  private static void drawPlayerPropertyHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, int scale, int faction, int distance)
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java]: This is nice.

File: src/UI/Art/SpriteArtist/MapSelectMenuArtist.java [hunk 3]
```diff
@@ -208,18 +208,21 @@ private static void drawPropertyNumber(Graphics g, int num, int x, int y)
...
-    do // We divide by 10 and truncate each time; expect three loops max.
-    {
-      int frame = num % 10;
-      g.drawImage(nums.getFrame(frame), x, y, numWidth, numHeight, null);
-      num /= 10; // Shift to the next higher digit in the number.
-      x -= numWidth; // Move the x-draw location to the left.
-    }while(num > 0);
+    BufferedImage numImage = SpriteUIUtils.getNumberAsImage(num);
+
+    g.drawImage(numImage, x, y, numImage.getWidth(), numImage.getHeight(), null);
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/MapSelectMenuArtist.java]: Somehow this is cutting off three digit property-count values on the map-select screen.

File: src/UI/Art/SpriteArtist/InfoView.java (hunk 1 above)
Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/InfoView.java]: Forgot it existed.

File: src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java (hunk 2 above)
Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java]: Thanks.
It was inspired by going "wait, which team was that, again?" on a 14 player map.

File: src/UI/Art/SpriteArtist/MapSelectMenuArtist.java (hunk 3 above)
Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/MapSelectMenuArtist.java]: `getNumberAsImage()` has an "if" in place of a "while".

**Pull request merged**